.github/workflows/operator-version-update.yaml

```diff
@@ -21,20 +21,20 @@ on:  # yamllint disable-line rule:truthy
   workflow_dispatch:
     inputs:
       csm-version:
-        description: 'CSM program version, ex: v1.12.0, v1.13.0, ...'
+        description: 'CSM program version, ex: v1.14.0, v1.15.0, ...'
         required: true
-      latest-version:
-        description: 'Latest operator version, ex: v1.7.0, v1.8.0, ...'
-        required: true
-      existing-version:
-        description: 'Existing operator version, ex: v1.6.0, 1.7.0, ...'
+      update-option:
+        description: 'Select the update flag, ex. "nightly" or "tag"'
```

Comment:
Can you elaborate/clarify more on the description as if someone is running this for the first time?